docs/adr/0013-interactive-templates-repo.md

```diff
@@ -0,0 +1,51 @@
+# 13. Use external repo for interactive code schema and templates
+Date: 2023-03-24
+
+## Status
+Accepted
+
+## Context
+
+We need to support dynamically generating code from user input as part of OSI.
+Functionally, this has two main parts: 1) the form the user fills in, and 2 analysis
+code templates to be rendered with the form data.
+
+These two parts are tightly coupled - changes to the form data schema needs to
+match changes in the templates.
+
+However, whilst the form itself must live in job-server repo, there are reasons
```

Comment:
Might be worth linking to [this ADR](https://github.com/opensafely-core/job-server/blob/5197171e19de658f513156c3508b6cf5a1c55ed6/docs/adr/0005-migrate-interactive-into-job-server.md) here.